### DISCUSSION TITLE
[Feature] Ability to create album with photos that wont show up on main photo view

### DISCUSSION THREAD

**Discussion by Puupuls** [immich-app/immich#8616] [Feature Request]
### The feature

I love the possibilities of this app and that it feels like google, but I get to keep my files to myself.

What I would love and something that I feel many who take pictures both for personal and professional use would love is the ability to create a sharable album, e.g. after a photo shoot, that you can nicely share but that won't show up on your main photos feed, memories, persons etc

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #5158
